store/src/test/java/org/apache/rocketmq/store/MultiPathMappedFileQueueTest.java

```diff
@@ -83,18 +84,18 @@ public void testUpdatePathsOnline() {
 
         MessageStoreConfig config = new MessageStoreConfig();
         config.setMultiCommitLogPathEnable(true);
-        config.setCommitLogStorePaths("target/unit_test_store/a/:target/unit_test_store/b/:target/unit_test_store/c/");
-        MappedFileQueue mappedFileQueue = new MultiPathMappedFileQueue(config, 1024, null);
+        config.setCommitLogStorePaths("target/unit_test_store/a/;target/unit_test_store/b/;target/unit_test_store/c/");
```

Comment:
"," is more commonly used as the path separator in config parameters. IMHO, I suggest using "," instead of ";" as the path separator.